apps/common-app/src/examples/RuntimeTests/ReanimatedRuntimeTestsRunner/TestRunner.ts

```diff
@@ -36,15 +37,42 @@ function notifyJS(name: string) {
   notificationRegistry[name] = true;
 }
 
+export class RenderLock {
+  private _renderLock: LockObject = { lock: false };
+
+  public lockRender() {
```

Comment:
we don't need to duplicate a name
```suggestion
  public lock() {
```